spec/stat_tracker_spec.rb

```diff
@@ -35,4 +35,140 @@
     #there should be at least one game_team
     expect(@stat_tracker.game_teams.count).to be > 0  
   end
-end
+
+  describe '#highest_total_score' do
```

Comment:
All tests are looking accounted for and matching the spec harness, we're good to move onto our 'season statistics'. Great work!